config.yaml

```diff
@@ -178,6 +178,24 @@ commands:
 
   N98\Magento\Command\Installer\InstallCommand:
     magento-packages:
+      - name: magento-ce-1.9.3.4
```

Comment:
This name is reserved for original Magento CE packages by the vendor, not OpenMage as used in this entry.